### PR TITLE
Update #187 add gender breakouts for employment variables

### PR DESCRIPTION
**Pull request by maggiesu0725**

### CODE REVIEW

**Comment by maggiesu0725** [comment]
@lhendey quite a lot of variables added, for review

File: Prog/SF_2015_19/ACS_2015_19_wv_sum_all.sas [hunk 1]
```diff
@@ -12,7 +12,7 @@
  Modifications:
 **************************************************************************/
 
-%include "\\sas1\DCdata\SAS\Inc\StdLocal.sas";
+%include "L:\SAS\Inc\StdLocal.sas";
```
Comment (lhendey) [inline Prog/SF_2015_19/ACS_2015_19_wv_sum_all.sas:15]: need to get rid of this change. should be the sas1.

File: Macros/ACS_summary_geo_source_tr_vars.sas [hunk 2]
```diff
@@ -2143,6 +2412,44 @@
           B20005Ge57, B20005Ge58, B20005Ge59, B20005Ge60, B20005Ge61, B20005Ge62, B20005Ge63, B20005Ge64, 
           B20005Ge65, B20005Ge66, B20005Ge67, B20005Ge68, B20005Ge69, B20005Ge70 );
 
+/*gender*/
+
+      PopWorkFTLT75KB_M_&_years. =
```
Comment (lhendey) [inline Macros/ACS_summary_geo_source_tr_vars.sas:2417]: These should be less than 75K not 75K+

File: Macros/ACS_summary_geo_source_tr_vars.sas [hunk 3]
```diff
@@ -2490,6 +2797,296 @@
...
+          B20005Gm5, B20005Gm28 );
+      mPopWorkEarnAIOM_F_&_years. = 
+        %moe_sum( var=B20005Dm52, B20005Dm75, 
+          B20005Em52, B20005Em75, 
+          B20005Cm52, B20005Cm75, 
+          B20005Fm52, B20005Fm75, 
+          B20005Gm52, B20005Gm75 );
+
+	  mPopWorkFTLT35KB_M_&_years. = 
+        %moe_sum( var=B20005Bm6, B20005Bm7, B20005Bm8, B20005Bm9, B20005Bm10, B20005Bm11,
```
Comment (lhendey) [inline Macros/ACS_summary_geo_source_tr_vars.sas:2977]: should include e16 and e17?

File: Macros/ACS_summary_geo_source_tr_vars.sas [hunk 4]
```diff
@@ -2059,6 +2250,84 @@
           B20005Ge51, B20005Ge53, B20005Ge54, B20005Ge55, B20005Ge56, B20005Ge57, 
           B20005Ge58, B20005Ge59, B20005Ge60, B20005Ge61, B20005Ge62, B20005Ge63, B20005Ge64 );
 
+
+/*gender*/
+	  PopWorkFTLT35KB_M_&_years. =
```
Comment (lhendey) [inline Macros/ACS_summary_geo_source_tr_vars.sas:2255]: these should income e16 & e17?

File: Macros/ACS_summary_geo_source_tr_vars.sas [hunk 5]
```diff
@@ -2059,6 +2250,84 @@
           B20005Ge51, B20005Ge53, B20005Ge54, B20005Ge55, B20005Ge56, B20005Ge57, 
           B20005Ge58, B20005Ge59, B20005Ge60, B20005Ge61, B20005Ge62, B20005Ge63, B20005Ge64 );
 
+
+/*gender*/
+	  PopWorkFTLT35KB_M_&_years. = 
+        sum(B20005Be6, B20005Be7, B20005Be8, B20005Be9, B20005Be10, B20005Be11,
+            B20005Be12, B20005Be13, B20005Be14, B20005Be15);
+
+	  PopWorkFTLT35KB_F_&_years. =
```
Comment (lhendey) [inline Macros/ACS_summary_geo_source_tr_vars.sas:2259]: is less than 35K these should include e63 & e64

**Review by lhendey — CHANGES_REQUESTED** [review]
On both bg & tract file need to add gender to the full pop vars as well (line 220 for example on bg file)
Need to edit revisions statements when you run programs Acs_sf_2015_19_DC.sas, MD, WV, VA, etc. 
Need  PopInCivLaborForce to correctly calculate the unemployment rate.

**Pull request merged**